llvm/test/CodeGen/Mips/2008-07-06-fadd64.ll

```diff
@@ -1,8 +1,18 @@
-; RUN: llc -march=mips -mattr=single-float  < %s | FileCheck %s
+; RUN: llc -march=mips -mcpu=mips32 -mattr=single-float -O2 < %s | FileCheck %s
+; RUN: llc -march=mips -mcpu=mips32r2 -mattr=single-float -O2 < %s | FileCheck %s
```

Comment:
Don't need -O2. The mattr is also missing +, and -march should be mtriple

Reply:
OK, applied, thanks!